lib/Github/Api/Repository/Invitations.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Github\Api\Repository;
+
+use Github\Api\AbstractApi;
+
+class Invitations extends AbstractApi
+{
+    public function all($username, $repository, array $params = [])
```

Comment:
Please add method docs to all methods in this class. See other APIs/endpoints for example.